tests/console/yast2_lan.pm

```diff
@@ -65,6 +65,16 @@ sub run {
     }
 
     assert_screen 'test-yast2_lan-1';
+
+    # Do not set hostname via DHCP - poo#66775
+    if (is_opensuse) {
```

Comment:
Seems that we need to add an import to use it.

Reply:
There is already: `use version_utils ':VERSION';`